packages/kbn-optimizer/limits.yml

```diff
@@ -107,3 +107,4 @@ pageLoadAssetSize:
   osquery: 107090
   fileUpload: 25664
   banners: 17946
+  cases: 102558
```

Comment:
Moving `cases` out of `securitySolution` is under a feature flag.  while the size of `cases` goes up,`securitySolution` will go down once the feature flag is removed.

Reply:
Ok! Maybe when the flag is removed and `cases` is ready, we can check if we can optimize the bundle size.